common/HGNCParser.py

```diff
@@ -83,7 +83,7 @@ def __init__(self):
 
     def _get_hgnc_data_from_json(self):
 
-        r = requests.get(Config.GENES_HGNC)
+        r = requests.get('http://ftp.ebi.ac.uk/pub/databases/genenames/new/json/hgnc_complete_set.json')
```

Comment:
Could we also import this from a config file instead of hardcoding?

Reply:
Ah, I see the config file is a YAML. Then perhaps not importing, but passing this as a command line parameter? Anyway, I don't think hardcoding is the best way to go

Reply:
@ireneisdoomed What about this one?

Reply:
+1 for adding that file as command line argument. The current implementation makes our data sources obscured.

Reply:
The URL is specified in the yaml file + new parameter in the PheWAS parser.
Commit`7855a3d`